README.md

```diff
@@ -10,7 +10,30 @@ The Kubernetes operator of Lumigo provides a one-click solution to monitoring Ku
 
 ### Installation
 
-Install the Lumigo Kubernetes operator in your Kubernets cluster with [helm](https://helm.sh/):
+Install the Lumigo Kubernetes operator in your Kubernets cluster with [helm](https://helm.sh/), with one of the following methods:
+
+#### Install & monitor namespaces
+
+The following command installs the operator and immediately applies monitoring to the specified namespaces, with traces or logs enabled or disabled as specified (defaulting to `true` for both):
+
+```sh
+echo "
+cluster:
+  name: <your cluster name>
+lumigoToken:
+  value: <your Lumigo token>
+monitoredNamespaces:
+  - namespace: my-namespace-1-traces-and-logs
+  - namespace: my-namespace-only-traces
+    loggingEnabled: false
+  - namespace: my-namespace-only-logs
+    tracingEnabled: false
+" | helm upgrade -i lumigo lumigo/lumigo-operator --namespace lumigo-system --create-namespace --values -
+```
```

Comment:
given that method, IMO we should provide instructions on how to remove/ add namespaces